apps/test/unit/netsim/NetSimRouterLogModal.js

```diff
@@ -19,131 +20,137 @@ describe("NetSimRouterLogModal", function () {
     NetSimTestUtils.initializeGlobalsToDefaultValues();
   });
 
-  describe("Log Mode", function () {
-    var modal, rootDiv, testShard, router, levelConfig;
+  testUtils.forEveryBooleanPermutation(useNewLogBrowser => {
```

Comment:
Only change here is to run most tests in this file both with the experiment enabled and disabled.  Recommended to view without whitespace changes.